src/fsharp/tast.fs

```diff
@@ -3109,7 +3398,7 @@ and CcuResolutionResult =
     | UnresolvedCcu of string
 
 /// Represents the information saved in the assembly signature data resource for an F# assembly
-and PickledModuleInfo =
+and PickledCcuInfo =
```

Comment:
mhm. I assume this is already established, but maybe something like CU would fit better. I wonnder where the second c comes from.